core/templates/domain/skill/skill-rights.model.ts

```diff
@@ -18,15 +18,17 @@
 
 export interface SkillRightsBackendDict {
   'can_edit_skill_description': boolean,
-  'skill_id': string
+  'skill_id': string | null
 }
 
 export class SkillRights {
-  _skillId: string;
+  // The '_skillId' can be null for an interstitial SkillRights object.
+  // Check function 'createInterstitialSkillRights' for reference.
```

Comment:
Please explain in more detail why it can be null and what a value of null actually means.

If you just refer to an existing impl, then that raises the question of whether the impl can be improved so that this is not the case. So it's not a sufficient explanation.

Reply:
Done